adapters/discord/src/types/gateway.ts

```diff
@@ -46,7 +46,7 @@ export enum GatewayOpcode {
   HEARTBEAT_ACK = 11,
 }
 
-/** https://discord.com/developers/docs/topics/gateway#gateway-intents */
+/** https://discord.com/developers/docs/topics/gateway-events#gateway-intents */
```

Comment:
Obviously this should be https://discord.com/developers/docs/topics/gateway#gateway-intents.

It seems that you have done some replacements incorrectly.